### ISSUE TITLE
weird shadow on floating

### ISSUE THREAD

**Issue by primalmotion** [hyprwm/Hyprland#1204]
Not sure if new, but I just noticed this weird bug. 

https://user-images.githubusercontent.com/208711/206878568-4567ce0b-5866-4137-9003-2270f0cfc6b0.mp4

**Comment by vaxerski** [comment]
lol

**Comment by vaxerski** [comment]
you sure its the shadow? what if you turn it off

**Comment by primalmotion** [comment]
I think it's damage tracking. when I record with wf-recorder with --no-damage the issue does not happen.

**Comment by primalmotion** [comment]
https://user-images.githubusercontent.com/208711/206879145-8d56745e-92bc-4d44-8a59-60554e1407c4.mp4

Yeah even without shadow

**Comment by vaxerski** [comment]
can you nail down the commit / config opt that causes this? maybe try disabling blur optimiz?

**Comment by primalmotion** [comment]
I think this has been a while, so bisecting would be hard. let me play with the blur option, seeing if I can find some config that does not show this

**Comment by vaxerski** [comment]
if it ever was okay, bisecting 1000 commits takes like 10 rounds. Not bad.

**Comment by primalmotion** [comment]
so without  blur at all: no issue

If I let blur_size and blur_pass to defaults values ( I have 8 and 3) it does not happen while moving a single window. however:


https://user-images.githubusercontent.com/208711/206879520-dfe0ab99-70ec-4825-b063-6030ea8374f2.mp4

**Comment by MahouShoujoMivutilde** [comment]
Check that your boolean values in config aren't set to like `2` or something.

I remember having something kinda similar looking on floating windows too, I've just blamed it on nvidia, but then at some point I discovered that I had `blur = 2` instead of appropriate boolean.


https://user-images.githubusercontent.com/14999778/206904990-af1a6867-5d89-4ed2-8dc0-02eb1a55697a.mp4


https://user-images.githubusercontent.com/14999778/206904991-2a8953ee-ef05-4c4e-a86c-650d4c70741a.mp4

**Comment by primalmotion** [comment]
Interesting. I've checked all my booleans, but none of them add a weird value

**Comment by vaxerski** [comment]
bop the config here (attach) and I'll see if I can repro.

**Comment by primalmotion** [comment]
```
monitor=eDP-1,preferred,10000x10000,1
monitor=desc:Dell Inc. DELL U2720Q 12PZP83,preferred,11920x9180,1.3
monitor=desc:Dell Inc. DELL U3419W GF2W5T2,preferred,9230x8560,1

monitor=desc:Dell Inc. DELL U2717D 67YGV74A661L,preferred,12670x8210,1
monitor=desc:Dell Inc. DELL U2717D 67YGV74A661L,transform,1

monitor=DP-2,preferred,12410x10841,1

exec = lid.sh

input {
	kb_layout =
	kb_variant =
	kb_model =
	kb_options = compose:caps
	kb_rules =
	follow_mouse = true
	accel_profile = flat
	touchpad {
		natural_scroll = true
		disable_while_typing = true
		drag_lock = true
	}
}

general {
	sensitivity = 1.0
	main_mod = SUPER
	layout = master
	gaps_in = 12
	gaps_out = 24
	border_size= 3
	no_border_on_floating = true
	source = ~/.config/hypr/color.conf
}

master {
	new_is_master = false
	no_gaps_when_only = true
}

dwindle {
	preserve_split = true
	no_gaps_when_only = true
	force_split = 2
}

gestures {
	workspace_swipe = true
	workspace_swipe_invert = true
	workspace_swipe_fingers = 4
 	workspace_swipe_distance = 1300
	workspace_swipe_cancel_ratio = 0.5
	workspace_swipe_min_speed_to_force = 20
	workspace_swipe_create_new = false
}

misc {
	 disable_hyprland_logo = true
	 no_vfr = false
	 animate_manual_resizes = true
	 enable_swallow = true
	 swallow_regex = ^(Alacritty)$
}

decoration {
	rounding = 8
	blur = true
	blur_size = 8
	blur_passes = 2
	blur_new_optimizations = true
	drop_shadow = true
	shadow_range = 32
	shadow_render_power = 18
	col.shadow = rgba(00000044)
}

animations {
	enabled = true
	animation = windows,1,3,default,slide
	animation = border,1,2,default
	animation = fade,1,2,default
	animation = workspaces,1,3,default
	animation = specialWorkspace,1,5,default,fade
}

blurls=notifications
```

the rest are bindings and windowrule on unaffacted applications, which I don't think are relevant

**Comment by vaxerski** [comment]
tanks

**Comment by vaxerski** [comment]
fixed in 80f58bc93f5812cf236f52ba9315aef54407b28c
